clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -1066,6 +1067,13 @@ public GroupCoordinatorMetrics(Metrics metrics, String metricGrpPrefix) {
                         "several failed re-trials until it succeeded"),
                 new CumulativeCount()
             );
+            this.successfulRebalanceSensor.add(
+                metrics.metricName("rebalance-total-time",
```

Comment:
I think this is a good to add, about the metric name, wdyt about `rebalance-latency-total` or `rebalance-latency-cumulated-total` instead? I felt it is more consistent with other namings.